docker/scirius/suricata/scripts/suri_reloader

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
```

Comment:
This is already part of scirius code so we don't need to duplicate.

Reply:
This version is quite different: I has been adapted to use the UNIX socket (according to the path in env variables) and different path have been adapted as well